_includes/header.html

```diff
@@ -14,20 +14,30 @@
 
     <!-- Schema protocol -->
     <meta itemprop="name" content="{{ site.title }}">
-    <meta itemprop="description" content="{{ site.meta }}">
     <meta itemprop="image" content="{{ prepend: site.baseurl | prepend: site.url }}/images/defund12.org.png">
 
     <!-- Facebook Open Graph protocol -->
     <meta property="og:title" content="{{ site.title }}" />
     <meta property="og:site_name" content="{{ site.title }}" />
-    <meta property="og:description" content="{{ site.meta }}" />
     <meta property="og:image" content="{{ prepend: site.baseurl | prepend: site.url }}/images/defund12.org.png">
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="{{ site.baseurl }}">
 
     <!-- Twitter card protocol -->
     <meta name="twitter:card" content="summary_large_image">
     <meta name="twitter:title" content="{{ site.title }}">
     <meta name="twitter:image" content="{{ prepend: site.baseurl | prepend: site.url }}/images/defund12.org.png">
+
+    <!-- Localized descriptions for all the above protocols -->
+    {% if page.title %}
```

Comment:
I'm not sure I like how it's not immediately obvious how page.title is used within site.meta_localized. I was thinking of doing something more like

```
meta: Send pre-populated localized emails to officials in

{% if page.title %}
 <meta property="og:description" content="{{ site.meta }} {{ page.title }}." />
  {% else %}
  <meta property="og:description" content="{{ site.meta }} your city." />
  {% endif %}
```

but that's kinda gross too...